src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler_WrongTranspile.cs

```diff
@@ -0,0 +1,5778 @@
+//// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Contains local changes to RegexCompiler.cs. This is WIP and won't compile yet. Included just for reference